.circleci/config.yml

```
@@ -153,6 +169,35 @@ jobs:
           path: /tmp/test-reports
       - store_artifacts:
           path: /tmp/test-reports
+  test-ui:
+    docker:
+      - image: circleci/node:10-browsers
+        environment:
+          JOBS: 2
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-deps-{{ .Branch }}-{{ checksum "ui/yarn.lock" }}
```

I'd drop `branch` checks and only use checksum - allows us to reuse cache across branches effectively considering we rarely change yarn.lock and have a lot of short lived branches.

```suggestion
            - v1-deps-{{ checksum "ui/yarn.lock" }}
```

the same apply for save.

---

I was shamelessly copying [this](https://kevin.pfefferle.co/2018/01/27/circleci-2-with-ember) but yeah, agreed!